hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/response/snapshot/OMSnapshotCreateResponse.java

```diff
@@ -77,9 +76,9 @@ public void addToDBBatch(OMMetadataManager omMetadataManager,
         key, snapshotInfo);
 
     // TODO: [SNAPSHOT] Move to createOmSnapshotCheckpoint and add table lock
-    // Remove all entries from renamedKeyTable
-    TableIterator<String, ? extends Table.KeyValue<String, OmKeyRenameInfo>>
-        iterator = omMetadataManager.getRenamedKeyTable().iterator();
+    // Remove all entries from snapshotRenamedKeyTable
+    TableIterator<String, ? extends Table.KeyValue<String, String>>
```

Comment:
This is not use change but can you please use `try-with-resources` to close the iterator?

Reply:
Done.